src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -45,7 +45,7 @@ internal class MSBuildForwardingAppWithoutLogging
             };
 
         private readonly IEnumerable<string> _msbuildRequiredParameters =
-            new List<string> { "-maxcpucount", "-verbosity:m" };
+            new List<string> { "-maxcpucount", "-verbosity:m", "-terminallogger:auto" };
```

Comment:
if the environment variable is used to disable TL, does that take precedence? what about if a user specifies `-tl:off` manually - what's the precedence for required args here vs user-supplied arguments here vs env vars?

Reply:
The environment variable is lower precedence than the command line, so it would be completely ignored if we put this change in as-is today.